### ISSUE TITLE
Peer review post :)!

### ISSUE THREAD

**Issue by LauraSuper** [CIEE-Living-Data-Project/Mini-LDP-Project#3]
Week three has all of these:

By the end of this week, you will have:

** note - the ❌s are explained in the last point of this list

An OSF account and project;
A Zotero account;
A Zotero collection with some references;  ❌
Zotero library linked to OSF;
A *.bib file;
An ORCid account;  ❌
A GitHub account;
A welcome page Wiki on your OSF project;
Another wiki page that describes your bare-bones plans (serving as a mini example pre-registration);
Git installed on your computer;  ❌
Git global credentials configured + PAT stored;  ❌
Been added to the LDP Organization on GitHub;
A GitHub repository inside the LDP organization;
Finished creating necessary folders (code, data, outputs, etc) and associated _README files within your RStudio project directory;
Committed and pushed these changes to GitHub;
Connected your GitHub repo to OSF;
Your local project (the one on your computer) synced with GitHub and connected to OSF;  ❌
A *.csl file; (find a csl file [here (Links to an external site.)](https://github.com/citation-style-language/styles)) and [instructions](https://canvas.ubc.ca/courses/97661/pages/finding-and-using-a-csl-file-citation-style) on how to access and use a CSL file in your markdown document
An initial script (*.R file) which downloads your data and uses groundhog to manage packages;
A *.RProj file;
A preregistration file on OSF;
Created a view-only link for your OSF project and send it to your peer;
Grade your peer’s project using this same checklist (excluding the items with a ❌) and submit the report on Canvas (give them a maximum of 10 points).
Week 03

Congratulations on initializing your fully reproducible project! Now on week 03 we will focus on version control and archiving, so next week we can debug and celebrate your progress! By the end of this week, you should have:

An *.Rmd file that will become your manuscript.
A .gitgnore file
The complete folder structure and respective files in each folder (data - raw and clean -, code, manuscript, outputs, etc. + README and LICENSE - at least)
Your remote repository should have branches that reflect different aspects of the development of your project (e.g., one for analysis, one for figures, one for writing…)
An open pull request (PR) from a development branch to your main branch, requesting a review from one of your colleagues.
A review in a colleague’s repository. Help your colleague to improve their project and code! Is there a more reproducible way to do what they are trying to do? Is there a better file naming convention they could be using? Use issues and contributions (you can fork their repo, make the suggested changes and make a PR). Review their open PRs. We’re better together!
Grade your peer’s project using this same checklist by opening an issue in their repository and writing in its description your final grade (maximum of 20 points) and a brief commentary justifying your decision.
Submit on Canvas the link to your issue by Friday, September 23.

**Comment by LauraSuper** [comment]
I have these and still cannot see Zotero with this link: https://osf.io/vty49/?view_only=be3ac3b0b3ee4f6eaa97bc91f14666ab

**Comment by LauraSuper** [comment]
Laura for Stephanie:
18.75/20 

- still having issues seeing Zotero from OSF page
- not much written, but I only did -0.25 this time Stephanie to make it less hard ;)! Thank you for the comment by email this morning, and hope this helps!

Given a couple places unclear if connected and not much filled in for the pre-registration:
•	An OSF account and project; Yes
•	A Zotero account; Not clear if connected, does not show up (-0.25)
•	A Zotero collection with some references;  ❌
•	Zotero library linked to OSF; does not show up (-0.25)
•	A *.bib file; Yes
•	An ORCid account;  ❌
•	A GitHub account; Yes
•	A welcome page Wiki on your OSF project; Yes
•	Another wiki page that describes your bare-bones plans (serving as a mini example pre-registration); List but not fully explained (-0.25)
•	Git installed on your computer;  ❌
•	Git global credentials configured + PAT stored;  ❌
•	Been added to the LDP Organization on GitHub; Yes
•	A GitHub repository inside the LDP organization; Yes
•	Finished creating necessary folders (code, data, outputs, etc) and associated _README files within your RStudio project directory; Yes
•	Committed and pushed these changes to GitHub; Yes
•	Connected your GitHub repo to OSF; Yes
•	Your local project (the one on your computer) synced with GitHub and connected to OSF;  ❌
•	A *.csl file; (find a csl file here (Links to an external site.)) and [instructions](https://canvas.ubc.ca/courses/97661/pages/finding-and-using-a-csl-file-citation-style) on how to access and use a CSL file in your markdown document Yes
•	An initial script (*.R file) which downloads your data and uses groundhog to manage packages; Yes – see comment from her that she tried.
•	A *.RProj file; Yes
•	A preregistration file on OSF; Yes but not much filled in (-0.25)
•	Created a view-only link for your OSF project and send it to your peer; Yes
Week Three

An *.Rmd file that will become your manuscript; - check

A .gitgnore file; - check

The complete folder structure and respective files in each folder (data - raw and clean -, code, manuscript, outputs, etc. + README and LICENSE - at least); - check
Not much written (-0.25)

Your remote repository should have branches that reflect different aspects of the development of your project (e.g., one for analysis, one for figures, one for writing…); -check

An open pull request (PR) from a development branch to your main branch, requesting a review from one of your colleagues; - check

A review in a colleague’s repository. Help your colleague to improve their project and code! Is there a more reproducible way to do what they are trying to do? Is there a better file naming convention they could be using? Use issues and contributions (you can fork their repo, make the suggested changes and make a PR). - Review their open PRs. We’re better together! -check

**Comment by LauraSuper** [comment]
Great and here is the updated mark then :).

Laura for Stephanie:
19/20

Fixed seeing Zotero from OSF page: https://osf.io/vty49/?view_only=be3ac3b0b3ee4f6eaa97bc91f14666ab
 Not much written, but I only did -0.5 this time Stephanie to make it less hard ;)! Thank you for the comment by email this morning, and hope this helps!

Given a couple places unclear if connected and not much filled in for the pre-registration:
• An OSF account and project; Yes
• A Zotero account; Not clear if connected, does not show up (-0.25)
• A Zotero collection with some references; ❌
• Zotero library linked to OSF; does not show up (-0.25)
• A *.bib file; Yes
• An ORCid account; ❌
• A GitHub account; Yes
• A welcome page Wiki on your OSF project; Yes
• Another wiki page that describes your bare-bones plans (serving as a mini example pre-registration); List but not fully explained (-0.5)
• Git installed on your computer; ❌
• Git global credentials configured + PAT stored; ❌
• Been added to the LDP Organization on GitHub; Yes
• A GitHub repository inside the LDP organization; Yes
• Finished creating necessary folders (code, data, outputs, etc) and associated _README files within your RStudio project directory; Yes
• Committed and pushed these changes to GitHub; Yes
• Connected your GitHub repo to OSF; Yes
• Your local project (the one on your computer) synced with GitHub and connected to OSF; ❌
• A .csl file; (find a csl file here (Links to an external site.)) and [instructions](https://canvas.ubc.ca/courses/97661/pages/finding-and-using-a-csl-file-citation-style) on how to access and use a CSL file in your markdown document Yes
• An initial script (.R file) which downloads your data and uses groundhog to manage packages; Yes – see comment from her that she tried.
• A *.RProj file; Yes
• A preregistration file on OSF; Yes but not much filled in (-0.5)
• Created a view-only link for your OSF project and send it to your peer; Yes
Week Three

An *.Rmd file that will become your manuscript; - check

A .gitgnore file; - check

The complete folder structure and respective files in each folder (data - raw and clean -, code, manuscript, outputs, etc. + README and LICENSE - at least); - check
Not much written (-0.25)

Your remote repository should have branches that reflect different aspects of the development of your project (e.g., one for analysis, one for figures, one for writing…); -check

An open pull request (PR) from a development branch to your main branch, requesting a review from one of your colleagues; - check

A review in a colleague’s repository. Help your colleague to improve their project and code! Is there a more reproducible way to do what they are trying to do? Is there a better file naming convention they could be using? Use issues and contributions (you can fork their repo, make the suggested changes and make a PR). - Review their open PRs. We’re better together! -check

**Comment by spflaman** [comment]
Thanks for the feedback Laura! I will update my stuff with more information where you said not much written to make it a more complete project :)

**Comment by LauraSuper** [comment]
Yes and thank you for letting me help you! I look forward to your feedback too :)!

**Comment by spflaman** [comment]
Hi Laura, I was wondering why you took -0.5 for this: 

The complete folder structure and respective files in each folder (data - raw and clean -, code, manuscript, outputs, etc. + README and LICENSE - at least); - check
Not much written (-0.5)

I had all the folders with respective files and I have stuff written in all my README and LICENCE files. The only files within the folder structures that don't have anything written in them is my raw and clean data files. I am just wondering exactly how much you are expecting for this? Just because this item doesn't explicitly say we need lots written within all these files. Thanks!
